### PR TITLE
add turbo tab to heroes section, following previous pull request feedback.

### PR DESCRIPTION
**Pull request by barbmich**

### CODE REVIEW

File: src/lang/en-US.json [hunk 1]
```diff
@@ -433,13 +433,18 @@
   "subheading_team_elo_rankings": "k=32, init=1000",
   "hero_pro_tab": "Professional",
   "hero_public_tab": "Public",
+  "hero_turbo_tab": "Turbo",
```
Comment (Argn0) [inline src/lang/en-US.json:436]: Pretty sure there is already a string for "Turbo".

File: src/components/Heroes/rankColumns.ts [hunk 2]
```diff
@@ -54,9 +55,34 @@ type Row = {
...
+    [HeroesTab.PUBLIC]: generatePublicTabColumns(props.strings),
+    [HeroesTab.TURBO]: generateTurboTabColumns(props.strings),
+  }
+  return columns[props.tabType];
+};
+
+const generateTurboTabColumns = (strings: GlobalString) => {
+  const heroColumn = generateHeroColumn(strings)
+
+  console.log(strings)
```
Comment (howardchung) [inline src/components/Heroes/rankColumns.ts:69]: remove this

File: src/lang/en-US.json (hunk 1 above)
Comment (howardchung) [reply src/lang/en-US.json:436]: I'm fine with this being separate.

**Comment by howardchung** [comment]
LGTM after removing the log call

**Comment by howardchung** [comment]
Preview link: https://deploy-preview-2706--opendota-staging.netlify.app/heroes/turbo

**Review by howardchung — APPROVED** [review]

**Pull request merged**